app/controllers/super_admins/agents_controller.rb

```diff
@@ -6,6 +6,8 @@ def sign_in_as
       if sign_in_as_allowed?
         sign_out(:user)
         bypass_sign_in(agent, scope: :agent)
+        # On ajoute une clé de session pour savoir qu’un super-admin s’est connecté en tant qu’agent
+        session[:sign_in_as] = true
```

Comment:
Le fait qu'on ai deux commentaires qui paraphrasent le code, c'est un peu un smell qui indique qu'on pourrait améliorer le naming pour éviter qu'ils soient nécessaire.
Peut être qu'en mettant un nom de clé plus explicite, on éviterait d'ajouter du bruit, par exemple : 
```suggestion
        session[:super_admin_signed_in_as_agent] = true
```

Reply:
Effectivement beaucoup plus clair !